tests/test_core.py

```diff
@@ -6510,6 +6510,7 @@ def test_large_exported_response(self):
   def test_add_function(self):
     self.set_setting('INVOKE_RUN', 0)
```

Comment:
Makes me think this should have been called "CALL_MAIN".

Reply:
Yeah, CALL_MAIN would have been a better name, I agree.